install_test_visibility.sh

```diff
@@ -54,6 +54,21 @@ install_java_tracer() {
   echo "DD_TRACER_VERSION_JAVA=$(command -v java >/dev/null 2>&1 && java -jar $filepath_tracer || unzip -p $filepath_tracer META-INF/MANIFEST.MF | grep -i implementation-version | cut -d' ' -f2)"
 }
 
+verify_checksum() {
```

Comment:
Would it be better to have a function that `which`es its way to finding the right executable? (I don't think this is a blocker though).

Reply:
I'll leave it as is for now, but thanks for the idea, might get back to it later